build-logic/build-update-utils/src/main/kotlin/gradlebuild/buildutils/tasks/GenerateSubprojectsInfo.kt

```diff
@@ -17,8 +17,10 @@
 package gradlebuild.buildutils.tasks
 
 import org.gradle.api.tasks.TaskAction
+import org.gradle.work.DisableCachingByDefault
 
 
+@DisableCachingByDefault(because = "Only filters the input artifact")
```

Comment:
Doesn't look like a filter to me. This is not even an artifact transform.

Reply:
Doesn't this need an `@OutputFile`? It seems to write to a file.

Reply:
Yeah, this should be `Not worth caching` and nothing about the filtering. The base class should probably declare the output file.

Reply:
Fixed.